scripts/run_frontend_cypress.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/env bash
+proj_dir=$(readlink -f `dirname $0`/..)
+
+pushd "$proj_dir/frontend"
+
+npx serve -s build -l 3000
```

Comment:
What does this do, by the way? Can / should we build & start the frontend using the Dockerfile, in a container?

Reply:
This spins up an adhoc HTTP file-server serving files from the folder `build` (similar to pythons `python3 -m http.server` ). I will add that to the documentation, that the frontend need to be build beforehand. `npx` download `serve` incase it was not installed. I wouldn't add `serve` as a dev dependency, since no actual dev dependencies are needed here. 

Using Docker poses an extra requirement towards the need to having Docker installed and thus having to invoke a `sudo`-action at least once. All other tools like maven, java, node usually don't need `sudo`.

So if you just want to test the app or run the cypress test, you couldn't do so in a restricted environment.

Reply:
> Using Docker poses an extra requirement towards the need to having Docker installed and thus having to invoke a sudo-action at least once. All other tools like maven, java, node usually don't need sudo.

Alright, I mean we should try to replicate a real world as close as possible but I'd also prefer a simple setup.

But then I'd vote to not add another dependency but just use `server.js`, which was written precicely to serve the built frontend from `./build`